imports/plugins/core/checkout/client/components/cartIcon.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component, PropTypes } from "react";
+import { Reaction } from "/client/api";
+
+class CartIcon extends Component {
+  static propTypes = {
+    cart: PropTypes.object
+  }
+
+  handleClick = (event) => {
+    event.preventDefault();
+    return $("#cart-drawer-container").fadeOut(300, function () {
```

Comment:
Updating this to use `Velocity` from `velocity-animate` instead of $:

```
  Velocity(cartDrawer, { opacity: 1 }, 300, () => {
    Reaction.toggleSession("displayCart");
  });
```